### PR TITLE
Add PopKorn multiplatform dependency injection

### PR DESCRIPTION
**Pull request by corbella83**
Adding my Multiplatform Dependency injection library (PopKorn) to this list.
library: 
     https://github.com/corbella83/PopKorn
example of multiplatform project using it (plus ktor, coroutines and kotlinx.serialization):
   https://github.com/corbella83/FullMultiplatform
medium article:
   https://medium.com/@corbella83/popkorn-kotlin-dependency-injection-90990203cba8

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -748,5 +748,10 @@
     "github": "Kodein-Framework/Kodein-DB",
     "category": "DataBase",
     "maven": "https://dl.bintray.com/kodein-framework/Kodein-DB/org/kodein/db/kodein-db/"
+  },
+  {
+    "github": "corbella83/PopKorn",
+    "category": "DI",
+    "maven": "https://repo1.maven.org/maven2/cc/popkorn/"
```
Comment (Alex009) [inline libraries.json:755]: please change address to `https://repo1.maven.org/maven2/cc/popkorn/popkorn/`

Comment (Alex009) [inline libraries.json:755]: please change address to `https://repo1.maven.org/maven2/cc/popkorn/popkorn/`

**Review by Alex009 — CHANGES_REQUESTED** [review]

File: libraries.json (hunk 1 above)
Comment (corbella83) [reply libraries.json]: Done!

**Review by Alex009 — APPROVED** [review]

**Comment by Alex009** [comment]
thanks!

**Pull request merged**